### DISCUSSION TITLE
Is it possible to add "Drop to Bed" icon?

### DISCUSSION THREAD

**Discussion by Dominik1980** [OrcaSlicer/OrcaSlicer#3307] [Ideas]
Dear programmers,
is it possilbe to add a icon like this, like in Prusaslicer:

![droptobed](https://github.com/SoftFever/OrcaSlicer/assets/49302019/8635ba68-b399-4e50-9aeb-51270ca780d6)

**Comment by djngnear** [comment]
+1 for me. I've needed this many times
